types/cordova-plugin-bluetoothclassic-serial/index.d.ts

```diff
@@ -0,0 +1,105 @@
+// Type definitions for cordova-plugin-bluetoothClassic-serial 0.9.8
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).